docs/guides/local-apt-repository.rst

```diff
@@ -69,15 +70,15 @@ dependency and that will kick off the entire build process for you.
 Expect it to take 5 to 15 minutes depending on how fast your server is. You only
 need to do this once.
 
-Where are the packages
+Where are the packages?
 ~~~~~~~~~~~~~~~~~~~~~~
 
 Good question, they have been transferred to your Ansible controller in the
 :file:`secret/reprepro/includedeb/wheezy-backports/` directory.
 
 At this point you can delete your build server.
 
-Configure the local APT server
+Configuring the local APT server
 ------------------------------
```

Comment:
The underline needs to match the header, otherwise it won't be parsed correctly by Sphinx.